### PR TITLE
Medical Cyborg Gripper Upgrade

### PR DESCRIPTION
**Pull request by Runian**
# Document the changes in your pull request
Adds an upgrade for medical cyborgs that gives a gripper that lets them hold bottles, beakers, and blood bags.
Similar cost of the Plasma Cutter Upgrade, but a few swapped materials: 3000 metal (1.5 sheets), 1000 glass (0.5 sheets), 2000 gold (1 sheet), 500 silver (0.25 sheets).

Locked behind Cyborg Upgrades: Medical

Gripped items now copy overlays in case they need it.

# Wiki Documentation
New upgrade for Cyborg Upgrades.
- Name: "medical cyborg gripper".
- Description: "A simple grasping tool for interacting with various medical related items."
- Cost: 3000 metal (1.5 sheets), 1000 glass (0.5 sheets), 2000 gold (1 sheet), 500 silver (0.25 sheets).

New item to add to Cyborg Upgrades: Medical
- Design name: "Cyborg Upgrade (Medical Gripper)"

# Changelog
:cl:  
rscadd: A new upgrade for medical cyborgs that gives them a gripper that lets them hold bottles, beakers, and blood bags.
tweak: Cyborg gripper now copy overlays for the held items.
/:cl:

### CODE REVIEW

**Comment by KingSuperBOB** [comment]
medborg finally able to do chemistry and viro?

**Comment by ju45he** [comment]
I can think of a billion things mediborgs can do if they're evil as of now, all we need is a syringe gun and the mediborg will be unstoppable.

File: code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -924,6 +934,5 @@
 		/obj/item/kitchen/knife/combat,
 		/obj/item/kitchen/knife/envy,
 		/obj/item/kitchen/knife/rainbowknife,
-		/obj/item/kitchen/knife/ritual
```
Comment (Moltijoe) [inline code/game/objects/items/robot/robot_items.dm]: Whats happening here?

Comment (Runian) [reply code/game/objects/items/robot/robot_items.dm]: Whoops. Was from resolving conflicts on web edit.

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**